subprojects/diagnostics/src/integTest/groovy/org/gradle/api/reporting/model/ModelReportIntegrationTest.groovy

```diff
@@ -220,6 +224,10 @@ model {
     // nb: specifically doesn't use the parsing fixture, so that the output is visualised
     //If you're changing this you will also need to change: src/snippets/modelRules/basicRuleSourcePlugin/basicRuleSourcePlugin-model-task.out
     def "displays a report in the correct format"() {
+        assumeFalse(
+            "Java 18 currently creates issues with System.out charset resulting in these tests failing",
+            OperatingSystem.current().isWindows() && JavaVersion.current().isCompatibleWith(JavaVersion.VERSION_18)
```

Comment:
Can we rely on the TestPreconditions here too? e.g., something like `TestPrecondition.WINDOWS.fulfilled && TestPrecondition.JDK18_OR_LATER.fulfilled` 

This makes it easier to find these things later

Reply:
These have been changed to `@Requires` as well.